js/docx/smoke/api_gradient_stop/to_json.js

```diff
@@ -0,0 +1,28 @@
+builder.CreateFile("docx");
+var oDocument = Api.GetDocument();
+var oParagraph = oDocument.GetElement(0);
+var oGs1 = Api.CreateGradientStop(Api.CreateRGBColor(255, 111, 61), 0);
+var oGs2 = Api.CreateGradientStop(Api.CreateRGBColor(255, 213, 191), 100000);
+var oFill = Api.CreateRadialGradientFill([oGs1, oGs2]);
+var jsonFill = oFill.ToJSON();
```

Comment:
Why do you write a test for the [ApiGradientStop](https://api.onlyoffice.com/docbuilder/textdocumentapi/apigradientstop/tojson) class object but convert the Fill object to JSON?

Reply:
Fix in the new commit.